src/sdb.c

```diff
@@ -175,7 +175,7 @@ SDB_API const char *sdb_const_get_len (Sdb* s, const char *key, int *vlen, ut32
 	if (s->fd == -1)
 		return NULL;
 	(void) cdb_findstart (&s->db);
-	if (cdb_findnext (&s->db, hash, key, keylen) < 1)
+	if (cdb_findnext (&s->db, sdb_hash (key), key, keylen) < 1)
```

Comment:
what if sdb_hash != hashtable hash thing? imho sdb should only permit 1 string hashing method and be consistent with it everywhere